python/uds.py

```diff
@@ -355,11 +355,13 @@ def send(self, msgs: List[bytes], delay: float = 0) -> None:
         self._recv_buffer()
 
 class IsoTpMessage():
-  def __init__(self, can_client: CanClient, timeout: float = 1, debug: bool = False, max_len: int = 8):
+  def __init__(self, can_client: CanClient, timeout: float = 1, debug: bool = False, max_len: int = 8, response_pending_timeout: float = 5):
```

Comment:
move `response_pending_timeout` up to `UdsClient` because this is a UDS concept (not ISO-TP)

Reply:
done, I hope I understood this correctly